### DISCUSSION TITLE
'pysplishsplash.Exec.SimulatorBase' object has no attribute 'VTK_EXPORT'

### DISCUSSION THREAD

**Discussion by chunleili** [InteractiveComputerGraphics/SPlisHSPlasH#187] [Q&A] [answered]
Hi
I want to run the following code in the documentation. It intends to run the python binding with VTK output.
![image](https://user-images.githubusercontent.com/48758868/174012901-f898e877-74a6-4a4d-975e-9fda1623219c.png)

```
import pysplishsplash as sph
from pysplishsplash.Extras import Scenes
import os

def main():
    base = sph.Exec.SimulatorBase()
    output_dir = os.path.abspath("where/you/want/the/data")
    base.init(useGui=False, outputDir=output_dir, sceneFile=Scenes.DoubleDamBreak)
    base.setValueFloat(base.STOP_AT, 20.0) # Important to have the dot to denote a float
    base.setValueBool(base.VTK_EXPORT, True)
    # Uncomment the next line to set the output FPS value (must be float)
    # base.setValueFloat(base.DATA_EXPORT_FPS, 10000.) 
    base.run()

if __name__ == "__main__":
    main()
```

However, I got this error.
![image](https://user-images.githubusercontent.com/48758868/174012809-8a85ed94-45b5-4914-801c-294c30d66d62.png)

**Comment by janbender** [comment, marked as answer]
The correct command is: 

base.activateExporter("VTK Exporter", True)

You can also take a look at the example: complex_scene.py

**Reply by chunleili** [reply]
Thanks!
One sugguestion: Maybe your documentation should be updated, especially the part about python binding...

**Reply by janbender** [reply]
It is on our long ToDo-list. However, the examples should give you a hint how to use the interface,

**Reply by chunleili** [reply]
> It is on our long ToDo-list. However, the examples should give you a hint how to use the interface,

Good! Looking forward to your progress.
